src/admin/paths/admin@employees.yaml

```diff
@@ -6,6 +6,7 @@ get:
   parameters:
     - $ref: ../../components/parameters/created_at.yaml
     - $ref: ../../components/parameters/updated_at.yaml
+    - $ref: ../../components/parameters/is_enabled.yaml
```

Comment:
what about status like pending_details ?

Reply:
we are deprecating pending_details in platform side